src/SW_Main_lib.c

```diff
@@ -194,6 +197,17 @@ void sw_init_args(int argc, char **argv,
                     }
 					break;
 
+                case 6: /* -s */
+                    *userSUID = atoi(str) - 1; /* Make suid base0 */
+
+                    if(*userSUID < 0) {

```

Comment:
Any value for "s" triggers the error "Input suid was less than 1. Please input a value >= 1."
e.g.,
`bin/SOILWAT2 -d tests/example/ -f files.in -s 1`